www/core/nightlies/next_minor_extension.xml

```diff
@@ -19,81 +19,4 @@
-		<downloads>
-			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-0-4/Joomla_4.0.4-Stable-Update_Package.zip</downloadurl>
-		</downloads>
-		<tags>
-			<tag>stable</tag>
-		</tags>
-		<supported_databases mysql="5.6" mariadb="10.1" postgresql="11.0" />
-		<php_minimum>7.2.5</php_minimum>
-		<maintainer>Joomla! Production Department</maintainer>
-		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<section>STS</section>
-		<targetplatform name="joomla" version="4.0" />
-	</update>
-	<update>
-		<name>Joomla! 4.4 Nightly Build</name>
-		<description>Joomla! CMS</description>
-		<element>joomla</element>
-		<type>file</type>
-		<version>4.4.1-dev</version>
-		<infourl title="Joomla! Nightly Builds">https://developer.joomla.org/nightly-builds.html</infourl>
-		<downloads>
-			<downloadurl type="full" format="zip">https://developer.joomla.org/nightlies/Joomla_4.4.1-dev-Development-Update_Package.zip</downloadurl>
-		</downloads>
-		<tags>
-			<tag>stable</tag>
-		</tags>
-		<supported_databases mysql="5.6" mariadb="10.1" postgresql="11.0" />
-		<php_minimum>7.2.5</php_minimum>
-		<maintainer>Joomla! Production Department</maintainer>
-		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<targetplatform name="joomla" version="4.0.[456]" />
-	</update>
-	<update>
-		<name>Joomla! 4.4 Nightly Build</name>
-		<description>Joomla! CMS</description>
-		<element>joomla</element>
-		<type>file</type>
-		<version>4.4.1-dev</version>
-		<infourl title="Joomla! Nightly Builds">https://developer.joomla.org/nightly-builds.html</infourl>
-		<downloads>
-			<downloadurl type="full" format="zip">https://developer.joomla.org/nightlies/Joomla_4.4.1-dev-Development-Update_Package.zip</downloadurl>
-		</downloads>
-		<tags>
-			<tag>stable</tag>
-		</tags>
-		<supported_databases mysql="5.6" mariadb="10.1" postgresql="11.0" />
-		<php_minimum>7.2.5</php_minimum>
-		<maintainer>Joomla! Production Department</maintainer>
-		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<targetplatform name="joomla" version="4.[1234]" />
-	</update>
-	<update>
-		<name>Joomla! 5.0 Nightly Build</name>
-		<description>Joomla! CMS</description>
-		<element>joomla</element>
-		<type>file</type>
-		<version>5.0.1-dev</version>
-		<infourl title="Joomla! Nightly Builds">https://developer.joomla.org/nightly-builds.html</infourl>
-		<downloads>
-			<downloadurl type="full" format="zip">https://developer.joomla.org/nightlies/Joomla_5.0.1-dev-Development-Update_Package.zip</downloadurl>
-		</downloads>
-		<tags>
-			<tag>stable</tag>
-		</tags>
-		<supported_databases mysql="8.0.13" mariadb="10.4" postgresql="12.0" />
-		<php_minimum>8.1.0</php_minimum>
-		<maintainer>Joomla! Production Department</maintainer>
-		<maintainerurl>https://www.joomla.org</maintainerurl>
-		<targetplatform name="joomla" version="5.[0]" />
-	</update>
```

Comment:
you removed the special handling for 4.0.x series at least I'm not sure what you are doing with it?

did you changed it to j4/default.xml?

Reply:
It's in minor.xml

Reply:
Yes, all handling of 4.0 is handled in j4/default.xml